### DISCUSSION TITLE
NDBC 1.1.2 on PyPI

### DISCUSSION THREAD

**Discussion by Weidav** [supermanzer/NDBC#42] [Q&A] [answered]
Hey guys,
I just started working with your library and was wondering if / or when version 1.1.2 will be released on PyPi.
Thanks,
David

**Comment by supermanzer** [comment, marked as answer]
Hi,

Thanks for pointing this out. I've added an issue, #44 , to address this. Your call follow it to get updated when the new version is published.

**Reply by Weidav** [reply]
thanks!
